commands.go

```diff
@@ -103,6 +104,23 @@ func (cmd *UIntCommand) ValueAsLit() string {
 	return fmt.Sprintf("%d", cmd.Value)
 }
 
+// DirectDeviceCommand executes the raw command passed to the ELM chipset 
+func (dev *Device) DirectDeviceCommand(directCmd string) (string, error) {
+        rawRes := dev.rawDevice.RunCommand(directCmd)
+
+        if rawRes.Failed() {
+                return "", rawRes.GetError()
+        }
+
+        if dev.outputDebug {
+                fmt.Println(rawRes.FormatOverview())
+        }
+
+        outputs := rawRes.GetOutputs()
+        output := outputs[0][:]
+        return strings.Trim(output, " "), nil
+}
+
```

Comment:
The file `commands.go` should only contain command types and their conversion functions. Since your function is actually a utility function of `Device`, it should be placed inside the file `device.go`. 

The function should also be renamed to fit in with the other command running functions `RunOBDCommand` and `RunManyOBDCommands`. Something like `RunRawOBDCommand` or `RunUntypedOBDCommand` is more suited. 

Otherwise it looks good :+1: